fileRead.py

```diff
@@ -1,3 +1,5 @@
+#This prpgram is for file reading!
+
 file = open("fileops.txt", "r")
 for name in file.readlines():
```

Comment:
for name in file.readlines(): -> for name in open("fileops.txt", "r").readlines():